src/status_im/contexts/profile/contact/contact_request/view.cljs

```diff
@@ -40,20 +50,26 @@
       (i18n/label :t/contact-request-message-prompt)]
      [rn/view {:style style/message-input-wrapper}
       [quo/input
-       {:type                :text
-        :multiline?          true
-        :char-limit          constants/contact-request-message-max-length
-        :max-length          constants/contact-request-message-max-length
-        :placeholder         (i18n/label :t/type-something)
-        :auto-focus          true
-        :accessibility-label :contact-request-message
-        :label               (i18n/label :t/message)
-        :on-change-text      on-message-change}]]
-     [quo/bottom-actions
-      {:actions          :one-action
-       :button-one-props {:disabled?           (string/blank? message)
-                          :accessibility-label :send-contact-request
-                          :customization-color customization-color
-                          :on-press            on-message-submit}
-       :button-one-label (i18n/label :t/send-contact-request)}]]))
+       {:type                  :text
+        :ref                   #(reset! input-ref %)
+        :multiline?            true
+        :char-limit            constants/contact-request-message-max-length
+        :max-length            constants/contact-request-message-max-length
+        :placeholder           (i18n/label :t/type-something)
+        :auto-focus            true
+        :accessibility-label   :contact-request-message
+        :label                 (i18n/label :t/message)
+        :on-change-text        on-message-change
+        :input-style           {:flex       1
+                                :min-height "100%"}
```

Comment:
if `:min-height` is `"100%"` doesn't that mean that we could set `:flex-shrink` as `0` instead?

IMO is a little strange to use  `:min-height` in that way, but sometimes styles are a little tricky,

Reply:
Yeah this was a bit strange, initially `min-height` was being used to force the input to take up the entire space inside of a view that had `flex-shrink: 1`, but I ended up removing this stuff because I found another of way of getting the input to properly grow and shrink.